src/features/Details/components/HeroDetailsList/HeroDetailsList.tsx

```diff
@@ -0,0 +1,64 @@
+import { jestIdsMap } from '@utils/jestHelpers';
+import { useMemo } from 'react';
+import styles from './HeroDetailsList.module.scss';
+
+interface HeroDetailsListProps<T> {
+  details: T;
+  detailNamesMap: Record<string, string>;
+}
+
+export const HeroDetailsList = <T,>({
+  details,
+  detailNamesMap,
+}: HeroDetailsListProps<T>) => {
+  const parsedDetails = useMemo(() => {
+    return Object.keys(details).reduce((accumulator, currentKey) => {
+      const value = String(details[currentKey as keyof T]);
+      const shouldSplit = value.match(/,(?![^(]*\))/);
```

Comment:
**suggestion** this regex is quite important, but you have chosen not to save it in a variable, but to copy and paste it into both occurrences, which may lead to mistakes in the future. 

Move regex to const and reuse it

Reply:
@kamil-borecki-cc True